recipe/meta.yaml

```diff
@@ -1,27 +1,26 @@
-{% set min_python = "python >=3.6" %}
-{% set version = "3.4.0" %}
-{% set min_python = "python >=3.6" %}
+{% set name = "gql" %}
+{% set version = "3.5.0" %}
 
 package:
   name: gql-split
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/g/gql/gql-{{ version }}.tar.gz
-  sha256: ca81aa8314fa88a8c57dd1ce34941278e0c352d762eb721edcba0387829ea7c0
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ccb9c5db543682b28f577069950488218ed65d4ac70bb03b6929aaadaf636de9
 
 build:
-  noarch: python
   number: 0
-  entry_points:
-    - gql-cli=gql.cli:gql_cli
+  skip: true  # [py<36 or s390x]
 
 requirements:
   host:
-    - {{ min_python }}
+    - python
     - pip
+    - wheel
+    - setuptools
   run:
-    - {{ min_python }}
+    - python
 
 test:
```

Comment:
Best to remove the mock test in the global output or replace it with a comment. It's not being run anyway.